lib/wallet/bench/api-bench.hs

```diff
@@ -528,6 +534,12 @@ mockNetworkLayer = dummyNetworkLayer
     , currentSlottingParameters = pure dummySlottingParameters
     , currentProtocolParameters = pure dummyProtocolParameters
     , currentNodeEra = pure $ Cardano.anyCardanoEra Cardano.BabbageEra
+    , currentNodeTip = pure BlockHeader
+        { slotNo = SlotNo 8888
+        , blockHeight = Quantity 9999
```

Comment:
I bet those hardcoded values are reasonable here

Reply:
My reasoning was the following:
*  A dummy value needed that is representative of a typical value;
* For an `Int` or `Natural`  the values `0`, `1`, `minBound`, `maxBound` aren't typical, so ruling them out;
* Something not too big and not too small would do it;
* Then its a matter of taste, `123456` would be reasonable too;

Reply:
In the real system, it's always true that the numerical values satisfy `slotNo > blockHeight`. I don't think that this will have any impact, but I think it's still worth to avoid giving a false impression in case someone reads this code. ("Wait, I once saw a code snippet where …")

Reply:
https://github.com/cardano-foundation/cardano-wallet/pull/4216